gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -243,14 +244,22 @@ public ManagedChannel createSingleChannel() throws IOException {
     return GrpcTransportChannel.create(outerChannel);
   }
 
-  private boolean isDirectPathEnabled() {
+  // TODO(weiranf): Use attemptDirectPath as the only indicator once setAttemptDirectPath is adapted
```

Comment:
please file a bug for this and reference it here

Reply:
Created b/192398509 to myself.